modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -15,19 +15,22 @@ Verify your image set configuration by performing a dry run without mirroring an
 +
 [source,terminal]
 ----
-$ oc mirror -c <image_set_config_yaml> --from file://<oc_mirror_workspace_path> docker://<mirror_registry_url> --dry-run --v2
+$ oc mirror -c <image_set_config_yaml> file://<oc_mirror_workspace_path> --dry-run --v2
 ----
++
 Where:
+
 - `<image_set_config_yaml>`: Use the image set configuration file that you just created.
```

Comment:
Also, the descriptions should be written similar to the docs guidelines where the sentence starts with "Specifies...":
```suggestion
- `<image_set_config_yaml>`: Specifies the image set configuration file that you created.
```